docs/csharp/programming-guide/unsafe-code-pointers/how-to-obtain-the-address-of-a-variable.md

```diff
@@ -29,7 +29,7 @@ int* p = &number; //address-of operator &
  You cannot get the address of a constant or a value.  
   
 ## Example  
- In this example, a pointer to `int`, `p`, is declared and assigned the address of an integer variable, `number`. The variable `number` is initialized as a result of the assignment to *p. If you make this assignment statement a comment, the initialization of the variable `number` will be removed, but no compile-time error is issued. Notice the use of the [Member Access](../../../csharp/programming-guide/unsafe-code-pointers/how-to-access-a-member-with-a-pointer.md) operator `->` to obtain and display the address stored in the pointer.  
+ In this example, a pointer to `int`, `p`, is declared and assigned the address of an integer variable, `number`. The variable `number` is initialized as a result of the assignment to *p. If you make this assignment statement a comment, the initialization of the variable `number` will be removed, but no compile-time error is issued.  
```

Comment:
The \* is interpreted as a formatting character, so the text appears in italics. Could you either escape it (\*p) or fence it, @pkulikov?

Reply:
@rpetrusha fixed!